src/osd/OSD.cc

```diff
@@ -9826,6 +9826,7 @@ const char** OSD::get_tracked_conf_keys() const
 {
   static const char* KEYS[] = {
     "osd_max_backfills",
+    "osd_max_remtoe_backfills",
```

Comment:
Typo here...

Reply:
no, i think max_remote_backfills is different from max_backfills in the many of cases;
there are two typical scenarios
especially the second point in cluster of All-hdd hard disks,  to improve data security.
1. in the case of three copies. if some pgs have only copy.  e.g. if we need backfill pg_3.5d pg_3.41 pg_3.63. the num lremote_reserver, we want, is different from num of remote_reserver
like this:
osd.0
local 3.5d
remote 3.63

osd.1
local  3.41
remote 3.5d

osd.2 
local  3.63
remote 3.5d
only 3.5d is backfilling

2. if the copies of some data is below min_size, we want speed up the recovery of this data and don't deal with other data(degraded) for the time being. so we need increase outgoing reservations of osd (acting set) osd and incoming reservations of osd (up set). the increase outgoing of acting osd will reduce the recovery speed of important data

Reply:
let me describe it.

Reply:
There's still a typo here that requires fixing or the config option won't be live-updatable, which seems to be your entire goal here.

Reply:
oh. sorry...

Reply:
@gregsfortytwo I fixed it.